static/js/checks.js

```diff
@@ -93,6 +93,34 @@ $(function () {
         $("#update-timeout-grace").val(rounded);
     });
 
+    var nagSlider = document.getElementById("nag-slider");
```

Comment:
Good implementation of the JS language in the UI

Reply:
Good implementation of  UIslider